src/main/java/org/broadinstitute/hellbender/tools/walkers/RevertBaseQualityScores.java

```diff
@@ -23,7 +23,7 @@
         programGroup = ReadProgramGroup.class
 )
 
-public class RevertBaseQualityScores extends ReadWalker {
+public final class RevertBaseQualityScores extends ReadWalker {
```

Comment:
Ah, I see. GSA policy was to avoid bundling unrelated changes in case something needs to be rolled back later down the road, because you wouldn't necessarily want e.g. these cleanup fixes to be undone (baby <=> bathwater). My personal preference would be to put the cleanup into a separate commit so you don't have to make a separate branch, but can still separate out the logical groupings of changes.